docs/can-guides/topics/debugging.md

```diff
@@ -197,14 +198,16 @@ new Observation(function fullName(){
 });
 ```
 
-> **Note:** If your function is a property on an observable map or list like [can-define/map/map],
+> **Note:** If your function is a property on an observable class like [can-observable-object],
 > you don't have to name it. For example, CanJS will name the `fullName` getter in the following example:
```

Comment:
Did you double check that this is true?

Reply:
Using this snippet:
```js
class MyEl extends StacheElement {
  static view = `
  {{ this.myProp }}
  <input value:bind="this.prop">
  `;
  
  static props = {
    prop: {
      type: type.convert(String),
      bind: fromAttribute
    },
    myProp: {
      get: function () {
        return this.prop
      }
    }
  };
}
customElements.define('my-el', MyEl);
```

When setting a breakpoint I see this output:
```
NOTIFY ran task: Observation<MyEl{}'s myProp getter>.onDependencyChange {fn: ƒ, context: my-el, args: Array(2), meta: {…}}
can-log.js:98 DERIVE ran task: Observation<MyEl{}'s myProp getter>.update
```
I'm assuming this is the expected right?

Reply:
Interesting. I assume this is because of the proxy? We should investigate.